gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/KspSubplugin.kt

```diff
@@ -367,13 +412,17 @@ class KspGradleSubplugin @Inject internal constructor(private val registry: Tool
                         kspTask.compilerPluginClasspath = project.configurations.getByName(pluginConfigurationName)
                         kspTask.commonSources.from(kotlinCompileTask.commonSources)
                         kspTask.compilerPluginOptions.addPluginArgument(kotlinCompileTask.compilerPluginOptions)
+                        val kspOptions = kspTask.options.get().flatMap { listOf("-P", it.toArg()) }
```

Comment:
Comment: this needs to happen after `configureAsKspTask` is called, where `kspTask.options` is set up.

Reply:
`configureAsKspTask` is at line 409 which is before this line?